chevah/keycert/ssh.py

```diff
@@ -299,7 +299,7 @@ def fromString(cls, data, type=None, passphrase=None):
             # Most probably some parts are missing from the key, so
             # we consider it too short.
             raise BadKeyError('Key is too short.')
-        except (struct.error,  binascii.Error, TypeError):
+        except (struct.error,  binascii.Error, TypeError, AssertionError):
```

Comment:
I was hoping that since we touch this part, we can dig deeper to find the root cause of this type of errors and try to fix it in a generic way so that we can have a nice keycert API which always return BadKeyError exception with valid text messages